inference-engine/src/mkldnn_plugin/utils/README.md

```diff
@@ -0,0 +1,73 @@
+# Debug capabilities
+Use the following cmake option to enable debug capabilities:
+
+`-DENABLE_CPU_DEBUG_CAPS=ON`
+
+## Blob dumping
+Blob dumping is controlled by environment variables (filters).
+
+The variables define conditions of the node which input, output and internal blobs
+should be dumped for.
+
+> **NOTE**: Nothing is dumped by default
+
+> **NOTE**: All specified filters should be matched in order blobs to be dumped
+
+Environment variables can be set per execution, for example:
+```sh
+    OV_CPU_BLOB_DUMP_DIR=dump_dir benchmark_app ...
```

Comment:
I am not sure we need `benchmark_app` here and in other example. I would suggest to remove

Reply:
I think without specifying binary file it will be less readable.
Do you propose to remove all the mentions of the binary?

Reply:
I do. Another solution: you can replace benchmark_app with abstract name. Both ok for me.

Reply:
Done